custom_components/kia_uvo/Vehicle.py

```diff
@@ -51,8 +54,13 @@ def __init__(
 
     async def update(self):
         try:
-            previous_vehicle_status = self.get_child_value("vehicleStatus")
-            previous_vehicle_location = self.get_child_value("vehicleLocation")
+            Vehicle.previous_vehicle_data = self.vehicle_data
```

Comment:
this will create long term issues when moving into `multiple vehicles` as this is a class variable rather than instance one. Can we convert this into instance variable?

Reply:
In my experience with using an instance variable with the Hyundai USA implementation I found that there either were multiple instances or that instances were garbage collected and reinitialized.  I mostly inferred this via reading the logs, so I could be wrong, but it seemed like different threads had different instances.